panda/src/putil/loaderOptions.h

```diff
@@ -34,6 +34,7 @@ class EXPCL_PANDA_PUTIL LoaderOptions {
     LF_no_cache          = 0x0030,  // no_disk + no_ram
     LF_cache_only        = 0x0040,  // fail if not in cache
     LF_allow_instance    = 0x0080,  // returned pointer might be shared
+    LF_no_filters        = 0x0100,  // disallow using texture pool filters
```

Comment:
Should this not be under TextureFlags?  Not sure of the implications one way or the other.